guides/v2.1/cloud/env/variables-post-deploy.md

```diff
@@ -15,14 +15,39 @@ stage:
 
 {% include cloud/customize-build-deploy.md %}
 
+### `TTFB_TESTED_PAGES`
+
+- **Default**— `[]` (an empty array)
+- **Version**—Magento 2.1.4 and later
+
+Configure _Time To First Byte_ (TTFB) testing for specified pages to test your {{ site.data.var.ece }} site performance. For each page to test, specify the path or the HTTP or HTTPS URL.
```

Comment:
"For each page to page to test, specify the absolute path reference or an absolute URL with protocol and host."